docs/local-development.md

```diff
@@ -690,6 +691,42 @@ index a390df2..c610c1a 100644
 +
 ```
 
+### Telemetry
+
+Telemetry is the Mozilla system for gathering usage metrics.
+The [Telemetry documentation][tel-docs] has in depth information, as well as a walk through of how
+to create new telemetry probes.
+
+There are two mechanisms available: Scalars, Histograms. Histograms are older, and Scalars is the
+new preferred method. However Scalars cannot do everything, so both are used.
+
+* **Scalars**: Count of an event
+* **Histograms**: Distribution of an event
+
+```js
+let loadSourceHistogram = Services.telemetry.getHistogramById(
+  "DEVTOOLS_DEBUGGER_LOAD_SOURCE"
+);
+loadSourceHistogram.add(delay) // time it took to load the source
+```
+
+```js
+Services.telemetry.scalarAdd("devtools.debugger.source_selected", 1);
+```
+
+When you add a new scalar or histogram, you'll need to add it to the
+[local histograms.json][localhistograms.json] or [local scalars.yml][localscalars.yaml]. This needs to be moved to MC,
```

Comment:
Documentation should no longer mention the local histrograms/scalars files. Can point to our current Bug as an example, and mention that the probes can be landed in mozilla-central before performing the implementation in the debugger.